plumbum.py

```diff
@@ -200,11 +195,19 @@ def main():
               .format(namespace))
         sys.exit(1)
 
-    print(template.render({
+    # base tokens
+    template_tokens = {
         'filters': filters,
         'region': region,       # Use for Auth config section if needed
         'resources': resources,
-    }))
+    }
+    # add tokens passed as cli args:
+    if tokens is not None:
+        for token_pair in tokens:
+            (key, value) = token_pair.split('=')
```

Comment:
does this need an input check, to make sure the user actually passed foo=bar tokens?